app/main.py

```diff
@@ -2,5 +2,16 @@
 
 
 def cache(func: Callable) -> Callable:
-    # Write your code here
-    pass
+    results = {}
+
+    def inner(*args) -> None:
+        nonlocal results
+        if args in results:
+            print("Getting from cache")
+        else:
+            print("Calculating new result")
+            results.update({args: func(*args)})
```

Comment:
```suggestion
            results.update[args] = func(*args)
```
No need to use .update if change only one key

Reply:
thanks!